arch/sim/src/Makefile

```diff
@@ -38,6 +38,7 @@ include $(TOPDIR)/Make.defs
 
 ARCH_SRCDIR = $(TOPDIR)$(DELIM)arch$(DELIM)$(CONFIG_ARCH)$(DELIM)src
 
+INCLUDES += ${shell $(INCDIR) "$(CC)" $(ARCH_SRCDIR)}
```

Comment:
$(ARCH_SRCDIR) would be $(TOPDIR)/arch/sim/src.  Why do you expect hostft.h to be there?

Reply:
hostfs.h is located at $(TOPDIR)/include/nuttx/hostfs.h

Or is there something else going on that is not clear?

Reply:
Okay, yes, I see in arch/sim/src/Makefile:

    213 hostfs.h: $(TOPDIR)/include/nuttx/fs/hostfs.h
    214         @echo "CP:  $<"
    215         $(Q) cp $< $@

Reply:
Yes, up_hostfs.c is built in host environment and then hostfs.h is copied locally for finding easlier.